<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>343</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>343</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>343</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>99</y>
     <width>95</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add -&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>54</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>50</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>77</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>73</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>95</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QListWidget" name="accountList">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>197</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>In Use :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Accounts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="inUseLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>197</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeAccount">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>126</y>
     <width>95</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="switchAccount">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>18</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>164</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Switch Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="notificationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 85, 127, 0);</string>
   </property>
   <property name="text">
    <string>Notifications</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>268</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>318</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Author: HyperDUDe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>246</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect on Starup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>18</y>
     <width>95</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoSwitch">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>225</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Switch</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="disableBalloonTips">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disable balloon tips</string>
   </property>
  </widget>
  <widget class="QSlider" name="switchTimeSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>193</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>164</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Minutes</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeAccount</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>switchAccount</tabstop>
  <tabstop>accountList</tabstop>
  <tabstop>logout</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
